src/NuGet.Services.EndToEnd/Support/Clients/GalleryClient.cs

```diff
@@ -156,6 +157,23 @@ public async Task UnlistAsync(string id, string version, ITestOutputHelper logge
             await SendToPackageAsync(HttpMethod.Delete, id, version, logger);
         }
 
+        public async Task SearchPackageODataV2FromDBAsync(string id, string normalizedVersion, ITestOutputHelper logger)
+        {
+            var galleryEndpoint = await GetGalleryUrlAsync(logger);
+            var url = $"{galleryEndpoint}/api/v2/Packages()?$filter=tolower(Id) eq '{id.ToLower()}' and NormalizedVersion eq '{normalizedVersion}' and 1 eq 1";
+            using (var httpClient = new HttpClient())
+            using (var request = new HttpRequestMessage(HttpMethod.Get, url))
+            {
+                using (var response = await httpClient.SendAsync(request))
+                {
+                    await response.EnsureSuccessStatusCodeOrLogAsync(url, logger);
+                    var responseContent = await response.Content.ReadAsStringAsync(); 
+                    logger.WriteLine($"The response to the request {url} is {responseContent}");
```

Comment:
`EnsureSuccessStatusCodeOrLogAsync` already logs the response

Reply:
Does the `responseContent` contain owner information? If yes, do we need to worry about GDPR data privacy?

Reply:
Removed the log line. The data is the test data, it is not customer data.